visualization/app/codeCharta/ui/artificialIntelligence/artificialIntelligence.component.html

```diff
@@ -43,8 +43,8 @@
 			<ul class="suspicious-metrics">
 				<li class="suspicious-metric" ng-repeat="item in $ctrl._viewModel.suspiciousMetricSuggestionLinks">
 					<md-button
-						class="metric-button {{ item.outlierCustomConfigId.length > 0 ? 'high-risk' : '' }}"
-						ng-click="$ctrl.applyCustomConfig(item.generalCustomConfigId)"
+						class="metric-button {{ item.isOutlier ? 'high-risk' : '' }}"
```

Comment:
Why do we need this css class (high-risk)?

Reply:
No, I will delete it.